transforms/remove-binding/index.js

```diff
@@ -0,0 +1,51 @@
+const { getParser } = require('codemod-cli').jscodeshift;
+
+module.exports = function transformer(file, api) {
+  const j = getParser(api);
+
+  const root = j(file.source);
+
+  const node = root.find(j.ObjectProperty, {
+    key: {
+      type: 'Identifier',
+    },
+  });
+
+  let filteredNodes = node.filter((path) => {
+    return path.value.key.name && path.value.key.name.endsWith('Binding');
+  });
+
+  let imports = root.find(j.ImportDeclaration);
+
+  let computedImports = imports.filter((importt) => {
+    return importt.value.source.value === '@ember/object/computed';
+  });
+
+  let alreadyHasAlias =
+    computedImports.length &&
+    computedImports
+      .get()
+      .node.specifiers.filter((specifier) => specifier.imported.name === 'alias');
+
+  if (!alreadyHasAlias[0] && filteredNodes.length) {
```

Comment:
💅🏽 if we want to treat this as a boolean (from what I see in L30), I would use `Array.some()`

```suggestion
    computedImports
      .get()
      .node.specifiers.some((specifier) => specifier.imported.name === 'alias');

  if (!alreadyHasAlias && filteredNodes.length) {
```